prow/config.yaml

```diff
@@ -15380,7 +15393,7 @@ periodics:
   agent: kubernetes
   name: istio-periodic-e2e-gke-addon
   labels:
-    preset-service-account: true
+    preset-istio-service: true
```

Comment:
can this do uploads to the GCS bucket?
/shrug

Reply:
/hold

Reply:
I'll whitelist that

Reply:
the job is busted anyway